core/storage/topic/gae_models.py

```diff
@@ -61,6 +61,8 @@ class TopicModel(base_models.VersionedModel):
     thumbnail_bg_color = ndb.StringProperty(indexed=True)
     # The description of the topic.
     description = ndb.TextProperty(indexed=False)
+    # The category this exploration belongs to.
```

Comment:
Docstring seems wrong -- "exploration"?

Also, what is your plan for updating existing topics in prod?

Reply:
Last time when I created a PR for adding description, we had a discussion that it's just 1 or 2 topics in the production right now and hence no job would be required. Have they increased recently? If yes, then I can write a One-off job for that.

Reply:
There's only 1 topic but we still need a way to handle that, right?

I'm not sure if you need a one-off job since this is a topic property. Just make sure there is some kind of upgrade pathway. I am fine with upgrading manually. But you should basically checkout develop, make a topic, switch to your branch (while keeping the dev server running) and ensure that the topic still loads without errors and that it's possible to change its description.

Reply:
Fixed the docstring. Done.